client/my-sites/drafts/author-selector.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+import NavSegmented from 'components/section-nav/segmented';
+import NavItem from 'components/section-nav/item';
+import userLib from 'lib/user';
+const user = userLib();
+
+function AuthorSelector( { selectedScope, onChange } ) {
+	const scopes = [
+		{
+			id: 'me',
+			caption: i18n.translate( 'Me', { context: 'Filter label for posts list' } ),
+			showGravatar: true
+		},
+		{
+			id: 'everyone',
+			caption: i18n.translate( 'Everyone', { context: 'Filter label for posts list' } ),
+			showGravatar: false
+		}
+	];
+
+	return (
+		<div className="drafts__author-selector">
+			<NavSegmented
+				label={ i18n.translate( 'Author', { context: 'Filter group label for segmented' } ) }
+			>
+				{ scopes.map( scope => {
+					const selectScope = () => onChange( scope.id );
```

Comment:
This is a violation of [the `jsx-no-bind` rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md), but the parser is not smart enough to identify it.

I made a comment about how to resolve a similar issue on another PR here: https://github.com/Automattic/wp-calypso/pull/8028#discussion_r82609215